plugins/backstage-plugin-github-insights/src/hooks/useRequest.ts

```diff
@@ -28,7 +28,7 @@ export const useRequest = (
   showTotal: boolean = false,
 ) => {
   const auth = useApi(githubAuthApiRef);
-  const { baseUrl } = useUrl();
+  const { baseUrl } = useEntityGithubScmIntegration();
```

Comment:
(Can't add a comment on the exact line)
Should we add the baseurl as part of the dependency array to the useAsync hook now if the 'loading' state exists when it is triggered. It would default to empty string until loading is done and these calls are done with `[]` deps array.